cloudbuild.yaml

```diff
@@ -13,10 +13,39 @@ steps:
         '--destination=us.gcr.io/$PROJECT_ID/chartcuterie:$COMMIT_SHA',
       ]
     timeout: 900s
+
   # Smoke test
   - name: 'us.gcr.io/$PROJECT_ID/chartcuterie:$COMMIT_SHA'
     args:
       - node
       - '/usr/src/app/lib/index.js'
       - '--help'
     timeout: 60s
+
+  - name: 'gcr.io/cloud-builders/docker'
+    secretEnv: ['DOCKER_PASSWORD']
+    entrypoint: 'bash'
+    args:
+    - '-e'
+    - '-c'
+    - |
+      # Only push to Docker Hub from master
+      [ "$BRANCH_NAME" != "master" ] && exit 0
+      docker push us.gcr.io/$PROJECT_ID/chartcuterie:nightly
```

Comment:
Why do we need this line? Don't think we need the `:nightly` tag on `us.gcr.io`

Reply:
We tag snuba and relay as nightlys on gcr, and use (at least relay) here from GCR https://github.com/getsentry/sentry/blob/46ae41c08642f9e5898fe41683edec615ca52675/src/sentry/conf/server.py#L1676

Reply:
snuba: https://github.com/getsentry/snuba/blob/master/cloudbuild.yaml#L66
relay: https://github.com/getsentry/relay/blob/master/cloudbuild.yaml#L84